ge08_team01/models/motorcycle_registry.py

```diff
@@ -0,0 +1,23 @@
+# Inherit the motorcycle.registry model and add the portal.mixin
+from odoo import fields, models
+
+
+class MotorcycleRegistry(models.Model):
+    _name = "motorcycle.registry"
+    _inherit = ["motorcycle.registry", "portal.mixin"]
+
+    is_public = fields.Boolean("Public", default=False)
+
+    def _get_portal_return_action(self):
+        self.ensure_one()
+        return self.env.ref("motorcycle_registry.registry_list_action")
+
+    # portal.mixin override
+    def _compute_access_url(self):
+        super()._compute_access_url()
```

Comment:
Do you need to call the super? It looks like you overwrite very record on self so it does not really matter what the super does right?